deployment/helm/values.yaml

```diff
@@ -50,7 +50,7 @@ status: Running
 
 # Expose Alert's User Interface
 exposeui: true
-exposedServiceType: NodePort # ClusterIP | NodePort | LoadBalancer
+exposedServiceType: NodePort # ClusterIP | NodePort | LoadBalancer | Openshift
```

Comment:
remove ClusterIP from it and also change Openshift to OpenShift

Reply:
Done!